docs/trace/building-your-own-processor/MyActivityProcessor.cs

```diff
@@ -1,50 +1,50 @@
-﻿// <copyright file="MyActivityProcessor.cs" company="OpenTelemetry Authors">
-// Copyright The OpenTelemetry Authors
-//
-// Licensed under the Apache License, Version 2.0 (the "License");
-// you may not use this file except in compliance with the License.
-// You may obtain a copy of the License at
-//
-//     http://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
-// </copyright>
-
-using System;
-using System.Diagnostics;
-using OpenTelemetry.Trace;
-
-internal class MyActivityProcessor : ActivityProcessor
-{
-    private readonly string name;
-
-    public MyActivityProcessor(string name)
-    {
-        this.name = name;
-    }
-
-    public override string ToString()
-    {
-        return $"{this.GetType()}({this.name})";
-    }
-
-    public override void OnEnd(Activity activity)
-    {
-        Console.WriteLine($"{this}.OnEnd");
-    }
-
-    protected override bool OnForceFlush(int timeoutMilliseconds)
-    {
-        Console.WriteLine($"{this}.OnForceFlush");
-        return true;
-    }
-
-    protected override void OnShutdown(int timeoutMilliseconds)
-    {
-        Console.WriteLine($"{this}.OnShutdown");
-    }
-}
+﻿// <copyright file="MyActivityProcessor.cs" company="OpenTelemetry Authors">
```

Comment:
Is there a CRLF/LF change? We can probably do all the normalization in one shot.

Reply:
yes. I am abandoning this PR anyway in favor of 1205